keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -565,7 +566,7 @@ feature, can it break the existing applications?).
 
 NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
 -->
-Yes.
+Yes, we can just disable the feature gate.
```

Comment:
L575: Have those tests been added?

Reply:
Yes, both are added, api side  and scheduling side. I'll update the document.